### ISSUE TITLE
Optgroup

### ISSUE THREAD

**Issue by SolusProject** [moreta/vue-search-select#58]
It would be nice to have option groups available, or maybe I just could not find it in the documentation.

**Comment by bbertucc** [comment]
I agree, an option group would give it much closer functionality to the HTML `<select>` element.
